docs/guide/rsun.rst

```diff
@@ -0,0 +1,70 @@
+.. _topic-guide-rsun:
+
+*********************************
+The role of ``rsun`` in ``sunpy``
+*********************************
```

Comment:
```suggestion
***************************
The role of "rsun" in sunpy
***************************
```

Reply:
For me, rsun is not code but a keyword, so I think it should like this and the changes below follow that approach.